src/components/Menu/Menu.stories.tsx

```diff
@@ -20,90 +21,167 @@ export default {
-        <Menu.Item
-          href="https://headlessui.com/react/menu#menu-button"
-          icon="link"
-        >
-          Headless UI Docs
-        </Menu.Item>
-        <Menu.Item
-          href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/menu"
-          icon="link"
-        >
-          MDN: Menu
-        </Menu.Item>
-        {/* eslint-disable-next-line no-alert */}
-        <Menu.Item onClick={() => alert('Item clicked')}>
-          Trigger Action
-        </Menu.Item>
-        <Menu.Item disabled href="https://example.org/" icon="warning">
-          Not Possible (disabled)
-        </Menu.Item>
-      </Menu.Items>
-    </Menu>
-  ),
+  args: {
+    children: (
+      <>
+        <Menu.Button>
+          Long Trigger Button Text to Demonstrate Popover Matching
+        </Menu.Button>
+        <Menu.Items data-testid="menu-content">
+          <Menu.Item
+            href="https://headlessui.com/react/menu#menu-button"
+            icon="link"
+          >
+            Headless UI Docs
+          </Menu.Item>
+          <Menu.Item
+            href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/menu"
+            icon="link"
+          >
+            MDN: Menu
+          </Menu.Item>
+          {/* eslint-disable-next-line no-alert */}
+          <Menu.Item onClick={() => alert('Item clicked')}>
+            Trigger Action
+          </Menu.Item>
+          <Menu.Item disabled href="https://example.org/" icon="warning">
+            Not Possible (disabled)
+          </Menu.Item>
+        </Menu.Items>
+      </>
+    ),
+  },
 };
 
 export const WithShortButtonText: StoryObj<MenuProps> = {
-  render: (args) => (
-    <Menu {...args}>
-      <Menu.Button>Menu</Menu.Button>
-      <Menu.Items data-testid="menu-content">
-        <Menu.Item
-          href="https://headlessui.com/react/menu#menu-button"
-          icon="link"
-        >
-          Headless UI Docs
-        </Menu.Item>
-        <Menu.Item
-          href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/menu"
-          icon="link"
+  args: {
+    children: (
+      <>
+        <Menu.Button>Menu</Menu.Button>
+        <Menu.Items data-testid="menu-content">
+          <Menu.Item
+            href="https://headlessui.com/react/menu#menu-button"
+            icon="link"
+          >
+            Headless UI Docs
+          </Menu.Item>
+          <Menu.Item
+            href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/menu"
+            icon="link"
+          >
+            MDN: Menu
+          </Menu.Item>
+          {/* eslint-disable-next-line no-alert */}
+          <Menu.Item onClick={() => alert('Item clicked')}>
+            Trigger Action
+          </Menu.Item>
+          <Menu.Item disabled href="https://example.org/" icon="warning">
+            Not Possible (disabled)
+          </Menu.Item>
+        </Menu.Items>
+      </>
+    ),
+  },
+};
+
+export const DotsVerticalButton: StoryObj<MenuProps> = {
```

Comment:
nit
```suggestion
export const WithVerticalDotsButton: StoryObj<MenuProps> = {
```

Reply:
@anniehu4 @jinlee93 we don't have a written convention for these but i have been using prefixes `With` if it demonstrates a composition, and `When` if the story exists to show some set of prop values.

Also, all `With` stories might actually be a signal that the story is actually a recipe in the format, "here's how you can combine ______ and _____ to get a _______"

Reply:
WithVerticalDots does sound like a recipe, and could be redundant as the WithCustomButton story. Consider removing?

Reply:
@jinlee93 the custom ones using `fpo` stuff make sense per component, as they show where placeholders belong. We can add the recipe showing how to use the vertical dots as/with a custom button and keep both

Reply:
Moved vertical dots story to its own recipe, since there already is a story using fpo block for button

Reply:
makes sense to me! would be good to document that convention somewhere